site/content/contribute/team_contributions/_index.md

```diff
@@ -9,22 +9,44 @@ Mattermost is an open source Slack-alternative that helps teams stay in sync, wi
 
 The project is developed by a global community of contributors and is maintained and supported by a distributed company, [Mattermost Inc.](https://about.mattermost.com/company/), based in Palo Alto, California.
 
+If you notice an error or omission in the list, please email community[at]mattermost.com so we can correct the issue.
+
+## Community Advocates
+
+Community Advocates by full name (first contribution date) are alphabetically listed below. The list is updated regularly (last updated June 26, 2020), and includes contributions to the mattermost.com/blog via an article.
+
+Join other community advocates by joining [our Mattermost Superstars program](https://mattermost.com/community/superstars).
```

Comment:
The link returns 404. Should it be setup before merging?

Reply:
Good call, I'll remove the link for now until the page is published